using-valtimo/access-control/backend-access-control.md

```diff
@@ -0,0 +1,114 @@
+# Backend-end access control
+This section describes how Policy Based Access Control (PBAC) can be configured in the backend.
+
+## Autodeployment
+PBAC can be configured in the backend using auto-deployment. 
+The deployment will scan for files on the classpath matching either `**/*.role.json` or `**/*.permission.json` for respectively role- and permission configurations.
+
+Every deployment file for PBAC represents a changeset, much like Liquibase. 
+These files contain a `changesetId` that should be unique over all deployment files that use changesets (currently only PBAC).
+
+The contents of a changeset cannot change as long as the `changesetId` does not. A change to an existing changeset can only be made when the `changesetId` also changes.
```

Comment:
I don't think we have documentation on changesets yet, which was added as part of this feature. I think this should be added as a separate page, and it should describe how to overwrite what's in the database with autodeployment.

Reply:
Created a new one just for this, and it'll also include updating this page.